packages/sw-routing/src/lib/express-route.js

```diff
@@ -50,16 +47,22 @@ import pathToRegExp from 'path-to-regexp';
  *
  * const router = new goog.routing.Router();
  * router.registerRoute({route});
+ *
+ * @memberof module:sw-routing
+ * @extends Route
  */
 class ExpressRoute extends Route {
   /**
-   * @param {string} path The path to use for routing.
+   * Constructor for ExpressRoute.
+   *
+   * @param {Object} input
+   * @param {string} input.path The path to use for routing.
    *        If the path contains [named parameters](https://github.com/pillarjs/path-to-regexp#named-parameters),
    *        then an Object mapping parameter names to the corresponding value
    *        will be passed to the handler via `params`.
-   * @param {Object} handler - An Object with a `handle` method. That method
-   *        will be used to respond to matching requests.
-   * @param {string} [method] Only match requests that use this
+   * @param {Object} input.handler - An Object with a `handle` method. That
+   *        method will be used to respond to matching requests.
```

Comment:
An Object with a `handle` method that will be used to respond to matching requests.

Combine the pieces into one.